lib/rubocop/cop/bundler/duplicated_group.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Bundler
+      # A Gem group should be listed only once in a Gemfile.
```

Comment:
Given the examples that's not exactly true, so you might want to expand this description.